.github/workflows/workflow.yml

```diff
@@ -0,0 +1,39 @@
+---
+name: Build
+# yamllint disable-line rule:truthy
+on:
+  push:
+  pull_request:
+
+jobs:
+
+  list-scenarios:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.listscenarios.outputs.scenarios }}
+    steps:
+      - uses: actions/checkout@v2
+      - id: listscenarios
+        uses: manics/action-ansible-molecule-list-scenarios@main
+
+  test:
+    name: Test
+    needs:
+      - list-scenarios
+    runs-on: ubuntu-latest
+    strategy:
+      # Keep running so we can see if other tests pass
+      fail-fast: false
+      matrix:
+        scenario: ${{fromJson(needs.list-scenarios.outputs.matrix)}}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          path: "${{ github.repository }}"
+      - name: molecule
+        uses: manics/action-ome-molecule@master
+        with:
+          scenario: ${{ matrix.scenario }}
+
+# notifications:
+#   webhooks: https://galaxy.ansible.com/api/v1/notifications/
```

Comment:
Have you found a generic way to notify Galaxy? I could also imagine this being a completely separate deployment workflow running on tags like "Publish on PyPI`.

Reply:
I haven't looked at galaxy notifications, it sounds like https://github.com/marketplace/actions/publish-ansible-role-to-galaxy mentioned in the comment you linked should work